### ISSUE TITLE
Semistandard tests failing - PR proposal

### ISSUE THREAD

**Issue by tyleralves** [Esri/esri-leaflet#825]
When I run `npm test`, `semistandard` fails with a couple formatting errors. I made the changes and am ready to submit a PR but it just seemed so bizarre that I figured I'd ask about it here. To address semistandard's problems I:

1. Added `if(error)` to log errors to console in FeatureManager.js
2. Removed line padding around a single `if else` statement in Request.js

```
> semistandard --verbose src/**/*.js | snazzy

semistandard: Semicolons For All! (https://github.com/Flet/semistandard)

\Users\Tyler\WebstormProjects\esri-leaflet\src\Layers\FeatureLayer\FeatureManager.js
  68:27  error  Expected error to be handled  handle-callback-err

\Users\Tyler\WebstormProjects\esri-leaflet\src\Request.js
  125:4  error  Block must not be padded by blank lines  padded-blocks

✖ 2 problems

```

**Comment by jgravois** [comment]
what version of semistandard?  i don't see a problem in `7.0.5`.

if you're going to go to the trouble of PRing to correct errors in a newer version, you might consider trying to bump us all the way up to [`8.0.0`](https://www.npmjs.com/package/semistandard).

**Comment by tyleralves** [comment]
I'm running 8.0.0.
I couldn't figure out why the handle-callback-err rule wasn't enforced in 7.0.5 but I was able to follow the breadcrumbs on the padded-blocks error (below). Will submit PR to bring up to semistandard 8.0.0.

padded-blocks error summary:
eslint-config-standard package `"padded-blocks": "never"` complains if there is an empty line at the beginning or end of an if-statement block. 

semistandard 7.0.5 depends on eslint-config-standard 4.0.0 so it doesn't include the `padded-blocks` rule per the below commit history:
- `padded-blocks` was removed from eslint-config-standard 3.8.0 [relax rule: padded-blocks](https://github.com/feross/standard/commit/6f75ec3c5ae1502b924f37ebb078d8)
- `padded-blocks` was re-added to eslint-config-standard 4.3.0 [Re-add rule: padded-blocks](https://github.com/feross/eslint-config-standard/commit/9166d52dc031e33f6de579db230c3a8c83233fe7)
